Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -618,19 +682,17 @@ CFStringRef _DWriteGetFamilyNameForFontName(CFStringRef fontName) {
 }
 
 /**
- * Private helper that parses a font name, and returns appropriate weight, stretch, and style values for DWrite accordingly
+ * Private helper that parses a font name, and returns appropriate weight, stretch, style, and family name values
  */
-static void __InitDWriteFontPropertiesFromName(CFStringRef fontName,
-                                               DWRITE_FONT_WEIGHT* weight,
-                                               DWRITE_FONT_STRETCH* stretch,
-                                               DWRITE_FONT_STYLE* style) {
+static void __InitDWriteFontPropertiesFromName(
+    CFStringRef fontName, DWRITE_FONT_WEIGHT* weight, DWRITE_FONT_STRETCH* stretch, DWRITE_FONT_STYLE* style, CFStringRef* familyName) {
     // Set some defaults for when weight/stretch/style are not mentioned in the name
     // Since this is a file-private helper, assume the pointers are safe to dereference.
     *weight = DWRITE_FONT_WEIGHT_NORMAL;
     *stretch = DWRITE_FONT_STRETCH_NORMAL;
     *style = DWRITE_FONT_STYLE_NORMAL;
 
-    CFStringRef familyName = _DWriteGetFamilyNameForFontName(fontName);
+    *familyName = _DWriteGetFamilyNameForFontName(fontName);
```

Comment:
>_DWriteGetFamilyNameForFontName [](start = 18, length = 31)

The string that is being returned here is not retained, how will the memory management work here?